ddtrace/tracer/option.go

```diff
@@ -133,6 +136,61 @@ func defaults(c *config) {
 			}
 		}
 	}
+
+	// Apply the user's options
+	for _, fn := range opts {
+		fn(c)
+	}
+
+	// If env, service, or version is unset, look for tags
```

Comment:
The way I remember commenting in Go (and what I've always used) aronud `if` statements and inline is always lower-case, and always inside the `if` statement explaining why when `env` is empty something else is happening.

e.g.
```
if c.env == "" {
    // fall back to global tags
    ...
```
Or something like that. What do you think? Do you think we should change this? Have you seen a different style elsewhere?

Reply:
I like the style you're describing. I think it's clearer than what I have here.